Removing 'very' with additional minor edits. (#1224)
RedisLabs/redislabs-docs: content/ri/memory-optimizations.md

> Combining small strings to Hashes reduces the memory used and in return save a cost.

Reviewer: ```suggestion
Combining small strings into hashes reduces the memory used and, in return, saves on costs.
```

> Hashes can be encoded to use memory efficiently, so Redis makers recommend that we use hashes whenever possible since "a few keys use a lot more memory than a single key containing a hash with a few fields", a key represents a Redis Object holds a lot more information than just its value, on the other hand a hash field only hold the value assigned, thus why it's much more efficient.

Reviewer: ```suggestion
```

Reviewer: This sentence can be safely removed altogether.

#### When to avoid converting hash to list

Avoid converting hashes to lists when: 

> 1. When your hash contains fewer than 50,000 objects.

Reviewer: ```suggestion
1. Your hash contains fewer than 50,000 objects.
```

> 1. When your hash contains fewer than 50,000 objects.

Reviewer: ```suggestion
1. When your hash contains fewer than 50,000 field-value pairs.
```

> 1. The size of your hash values are not consistent. Some values values contain a few fields while other contain many.

Reviewer: ```suggestion
1. The size of your hash values are not consistent (for instance, when some hashes contain only a few field-value pairs while others contain many).
```

### Switch to bloom filter or hyperloglog

Unique items can be difficult to count. Usually this means storing every unique item then recalling this information somehow. 

> Redis sets support this with a single command, however, storage and retrieval costs grow in proportion to the number of elements. If this becomes prohibative, consider HyperLogLog as a probabilistic alternative.

Reviewer: ```suggestion
Redis sets support this with a single command; however, storing every unique item you want want to count may use a prohibitive amount of memory. If this is the case, consider using a HyperLogLog instead. A HyperLogLog is a probabilistic data structure for counting unique items in a set. HyperLogLogs trade off perfect accuracy for less memory usage.
```

> Redis sets support this with a single command, however, storage and retrieval costs grow in proportion to the number of elements. If this becomes prohibative, consider HyperLogLog as a probabilistic alternative.

Reviewer: I've fixed the technical error in my revision below.
